src/installLogsPrinter.js

```diff
@@ -35,6 +35,7 @@ const LOG_SYMBOLS = (() => {
 })();
 
 let allMessages = {};
+let state = null;
```

Comment:
Better send the state on the other task as well.

Reply:
You mean remove that line, or null it on the first task as well?

Reply:
I mean dont have the variable here. Send always the state of the test in the task params.